server/src/repositories/album.repository.ts

```diff
@@ -203,7 +197,7 @@ export class AlbumRepository implements IAlbumRepository {
 
   @GenerateSql({ params: [DummyValue.UUID, [DummyValue.UUID]] })
   @Chunked({ paramIndex: 1 })
-  async removeAssets(albumId: string, assetIds: string[]): Promise<void> {
```

Comment:
Nit: I think I like `addAssets`/`removeAssets` better than `addAssetIds`/`removeAssetIds`.

Reply:
There is another one - get assets - which returns IDs. All three methods operate on IDs so it is a little weird to have them names differently imo.

Reply:
Ah, fair. `getAssetIds` is definitely better to distinguish between fetching the ID and the entity, and it's better for them to have consistent naming in that case.

Reply:
Yeah that was my thought process as well.